src/XrdHeaders.cmake

```diff
@@ -125,6 +125,18 @@ set( XROOTD_PRIVATE_HEADERS
   XrdOfs/XrdOfsHandle.hh
   XrdOfs/XrdOfsTrace.hh
   XrdOfs/XrdOfsTPCInfo.hh
+  XrdSsi/XrdSsiAtomics.hh
```

Comment:
Are we sure we don't want this in a separate package, say ssi-devel (together with *.so files) ?